kubernetes-deployment/src/helm_values.tf

```diff
@@ -0,0 +1,20 @@
+locals {
+  helm_values = {
+    commonLabels = var.md_metadata.default_tags
+    pod = {
+      annotations = {
+        "md-deployment-id" = lookup(module.application.params.md_metadata.deployment, "id", "")
+      }
+      labels = local.cloud_pod_labels[module.application.cloud]
```

Comment:
Does `mdxc` set a cloud? or should `module.application.cloud` be updates to `local.cloud`, which fetches from kubernetes cluster specs?

Reply:
Yeah, I think its an output from `mdxc`, and it is implied through the credentials provided. I think its best to leave it the way it is for now.